assets/src/settings-page/mobile-redirection.js

```diff
@@ -19,7 +19,8 @@ export function MobileRedirection() {
 
 	const { theme_support: themeSupport } = editedOptions || {};
 
-	if ( ! [ 'reader', 'transitional' ].includes( themeSupport ) ) {
+	// Don't show if the mode is standard or the themeSupport is not yet set.
+	if ( ! themeSupport || 'standard' === themeSupport ) {
```

Comment:
Why would `themeSupport` ever be empty? I suppose just when going through the wizard the first time?

Reply:
That's right. There is a technically a default mode when when `plugin_configured` is `false` we override `theme_support` with `null`.